### ISSUE TITLE
How to import videojs-vr with "import video-js from..."  to use it in webpack?

### ISSUE THREAD

**Issue by issimissimo** [videojs/videojs-vr#261]
Hello, this is not an issue but I don't know where to post it.
How can I import videojs-vr as ES6 class?

For example, I import "videojs" in this way: 
import videojs from 'video.js'

How can I do the same with videojs-vr?

Many thanks!

**Comment by bencook9** [comment]
Did you ever figure this out?

**Comment by gkatsev** [comment]
you should be able to do

```js
import videojs from 'video.js';
import 'videojs-vr';
```

If it doesn't work, you may have some unsual webpack config. If you can create a codesandbox.io minimal test case, we can take a look.

**Comment by bencook9** [comment]
This is my first time using codesandbox.io but here: https://codesandbox.io/s/videojs-vr-error-yuwbb4?file=/src/app/app.component.html

Let me know if this does not work for you.

So the main content of what I want to work is in the video-player folder. In the video-player.component.ts file I import videojs-vr. That seems to work but then I get an error on line 28 that says "Property 'vr' does not exist on type 'VideoJsPlayer'." I have been unable to resolve this error.

**Comment by gkatsev** [comment]
@bencook9 looks like you're having a typing issue. Unfortuantely, I don't think that there's types for videojs+video.js VR. However, if you remove the extra `require`, it makes it work. I added a `ts-ignore` so that typescript won't complain (also, not sure why styles we're being included, so, I manually included them in the component html).

https://codesandbox.io/s/videojs-vr-error-forked-xiw8up?file=/src/app/video-player/video-player.component.ts
